packages/twenty-server/src/engine/metadata-modules/field-metadata/field-metadata.service.ts

```diff
@@ -884,7 +887,7 @@ export class FieldMetadataService extends TypeOrmQueryService<FieldMetadataEntit
             ("fieldMetadataId", "position", "isVisible", "size", "viewId")
             VALUES ('${createdFieldMetadata.id}', '${
               lastPosition + 1
-            }', true, 180, '${view[0].id}')`,
+            }', ${isVisible}, 180, '${view[0].id}')`,
```

Comment:
User can still manually add more than 30 fields in view right? I don't see any error message to prevent that. Maybe I'm missing the root cause of this issue and this is not needed!